pkg/executor/traffic.go

```diff
@@ -202,7 +248,7 @@ func getTiProxyAddrs(ctx context.Context) ([]string, error) {
 }
 
 func requestOne(method, addr, path string, rd io.Reader) (string, error) {
-	url := fmt.Sprintf("%s://%s/%s", util.InternalHTTPSchema(), addr, path)
+	url := fmt.Sprintf("%s://%s%s", util.InternalHTTPSchema(), addr, path)
```

Comment:
Why changing removing the slash here?

Reply:
The leading slash is removed from the `path`.